cmd/nephe-controller/main.go

```diff
@@ -15,6 +15,7 @@
 package main
 
 import (
+	"antrea.io/nephe/pkg/cloud-provider/securitygroup"
```

Comment:
nit: order imports

Reply:
Solved

Reply:
Sorting meaning, have standard imports in beginning, then new line, then third party imports and then new line and finally antrea, and nephe imports.. Here you have antrea import in the beginning

Reply:
Solved